components/landing-page/places/index.js

```diff
@@ -0,0 +1,27 @@
+import SectionTitle from "../../utility/section_title"
+import { cities } from "../../data/cities";
+import { PlacesWrapper, PlacesFlex, PlacesTitle, PlacesSubTitle } from "./style";
```

Comment:
I like what you did here.....nice job. Do you get why i like using styled components. It removes reduces the number of html tags visible in your code even though it is the same tags, you can name the html tag anything you want.